edkahara/iReporter
# Ft create v2 logout endpoint #162650787

__What does this PR do__
Create a logout endpoint for v2 of iReporter.

__Description of tasks to be completed.__
Create an api endpoint that facilitates the logging out of a user. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-logout-endpoint-#162650787 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162650787
